AppController/djinn.rb

```diff
@@ -1475,11 +1493,19 @@ def set_property(property_name, property_value, secret)
         unless is_cloud?
           Djinn.log_warn("min_images is not used in non-cloud infrastructures.")
         end
+        if Integer(val) < Integer(@options['min_images'])
+          Djinn.log_warn("Invalid input: cannot lower min_images!")
+          return "min_images cannot be less than the defined layout."
```

Comment:
Please change the message to something like:
`min_images cannot be less than the nodes defined in ips_layout.`.

to make it clearer.